micrometer/src/main/java/feign/micrometer/MeteredClient.java

```diff
@@ -45,38 +45,82 @@ public MeteredClient(Client client, MeterRegistry meterRegistry, MetricName metr
   @Override
   public Response execute(Request request, Options options) throws IOException {
     final RequestTemplate template = request.requestTemplate();
-
+    final Timer.Sample sample = Timer.start(meterRegistry);
```

Comment:
can we also have a `finally` block, just in case?

Reply:
Done.